src/cmd/bundle.go

```diff
@@ -0,0 +1,171 @@
+
+	"github.com/defenseunicorns/zarf/src/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+var bundleCmd = &cobra.Command{
+	Use:     "bundle",
+	Aliases: []string{"b"},
+	Short:   lang.CmdBundleShort,
+}
+
+var bundleCreateCmd = &cobra.Command{
+	Use:     "create [DIRECTORY]",
+	Aliases: []string{"c"},
+	Args:    cobra.ExactArgs(1),
+	Short:   lang.CmdBundleCreateShort,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if !utils.IsDir(args[0]) {
+			return fmt.Errorf("first argument must be a valid path to a directory")
+		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		bndlConfig.CreateOpts.SourceDirectory = args[0]
+
+		bndlConfig.CreateOpts.SetVariables = bundler.MergeVariables(v.GetStringMapString(V_PKG_CREATE_SET), bndlConfig.CreateOpts.SetVariables)
+
+		bndlClient := bundler.NewOrDie(&bndlConfig)
+		defer bndlClient.ClearPaths()
+
+		if err := bndlClient.Create(); err != nil {
+			message.Fatalf(err, "Failed to create bundle: %s", err.Error())
+		}
+	},
+}
+
+var bundleDeployCmd = &cobra.Command{
+	Use:     "deploy [BUNDLE]",
+	Aliases: []string{"d"},
+	Short:   lang.CmdBundleDeployShort,
+	Args:    cobra.ExactArgs(1),
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if !utils.IsOCIURL(args[0]) && !bundler.IsValidTarballPath(args[0]) {
+			return fmt.Errorf("first argument must either be a valid OCI URL or a valid path to a bundle tarball")
+		}
+		return oci.ValidateReference(args[0])
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		bndlConfig.DeployOpts.Source = args[0]
+
+		bndlConfig.DeployOpts.SetVariables = bundler.MergeVariables(v.GetStringMapString(V_PKG_DEPLOY_SET), bndlConfig.DeployOpts.SetVariables)
+
+		bndlClient := bundler.NewOrDie(&bndlConfig)
+		defer bndlClient.ClearPaths()
+
+		if err := bndlClient.Deploy(); err != nil {
+			message.Fatalf(err, "Failed to deploy bundle: %s", err.Error())
+		}
+	},
+}
+
+var bundleInspectCmd = &cobra.Command{
+	Use:     "inspect [BUNDLE]",
+	Aliases: []string{"i"},
+	Short:   lang.CmdBundleInspectShort,
+	Args:    cobra.ExactArgs(1),
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if !utils.IsOCIURL(args[0]) && !bundler.IsValidTarballPath(args[0]) {
+			return fmt.Errorf("first argument must either be a valid OCI URL or a valid path to a bundle tarball")
+		}
+		return oci.ValidateReference(args[0])
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		bndlConfig.InspectOpts.Source = args[0]
+
+		bndlClient := bundler.NewOrDie(&bndlConfig)
+		defer bndlClient.ClearPaths()
+
+		if err := bndlClient.Inspect(); err != nil {
+			message.Fatalf(err, "Failed to inspect bundle: %s", err.Error())
+		}
+	},
+}
+
+var bundleRemoveCmd = &cobra.Command{
+	Use:     "remove [BUNDLE_NAME|BUNDLE_TARBALL|OCI_REF]",
+	Aliases: []string{"u"},
+	Args:    cobra.ExactArgs(1),
+	Short:   lang.CmdBundleRemoveShort,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if !utils.IsOCIURL(args[0]) && utils.InvalidPath(args[0]) {
```

Comment:
(case in point to the above, not sure why this line uses a different InvalidPath fuction)